### ISSUE TITLE
How to set up reverse proxy in front of naiveproxy

### ISSUE THREAD

**Issue by rocwhite123** [klzgrad/naiveproxy#557]
I'm using the [pocat/naiveproxy](https://hub.docker.com/r/pocat/naiveproxy) docker image to set up naiveproxy, and it works fine by itself. My question is whether I can put [nginx-proxy](https://hub.docker.com/r/nginxproxy/nginx-proxy) in front so that I can host other services on my server on port 443. What I want to achieve is to direct traffic addressed to a particular domain name (e.g., my-domain1.com) to naiveproxy while traffic addressed to other domains (e.g., my-domain2.com) gets directed to other docker containers. Is this possible (using either nginx, haproxy, or caddy)?

What I found is that once we have this additional layer (nginx-proxy), accessing my-domain1.com using a browser correctly sees the static files, but accessing my-domain1.com as a proxy fails.

**Comment by klzgrad** [comment]
It's possible, because this is all centered around fronting with a well-known reverse proxy. I don't know about your specific setup. And it's not my job to know.

**Comment by rocwhite123** [comment]
What a nice way to answer all user questions.

**Comment by zedifen** [comment]
Hi @rocwhite123, I understand that you needed help, but the developer's duty is to fix certain issues, not being a question answer machine. 😇 It's like that you won't ask a cook 👨‍🍳 to also feed the dishes to you. 😜

**Comment by rocwhite123** [comment]
Of course, this is a volunteer project so I was not expecting to be held hands. But one could stop at the first sentence, or if taking the effort to type a second sentence, simply mentioning layer 4 SNI routing is much more productive.

I got that information from a different project. So yes, there are more to a human being than technical prowess. Being nice probably helps lower blood pressure, creating a vibrant community, and who knows what else. But of course, if being confrontational is the vibe that makes one happy, I'm happy to oblige..

**Comment by klzgrad** [comment]
I'll let you know that environmental configuration is considered irrelevant to this project. If that information is immediately available or known to me, even if it is irrelevant, I'm not withholding it. But it is not known to me, and you are asking me to spend time to RTFM for you, and that I cannot do.

**Comment by rocwhite123** [comment]
No, I did not ask you to RTFM. My second post merely suggested that you could simply do away with the useless, confrontational second sentence in your original reply. Why not give it a try? People would respect you more and it's better for your health.

**Comment by klzgrad** [comment]
No, those are the two facts you need to know: 1) I don't know the answer to your question; 2) It is not my job to answer the kind of questions so don't waste time ask again, which I explain further what kind of questions won't be answered.

**Comment by rocwhite123** [comment]
Is it English or what? I clearly already get the two facts you needed me to know, as well as the solution to my original question..
